src/Fable.Transforms/Fable.Transforms.fsproj

```diff
@@ -49,7 +49,8 @@
     <ProjectReference Include="Rust/AST/Rust.AST.fsproj" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="../../lib/fcs/FSharp.Core.dll" />
-    <Reference Include="../../lib/fcs/FSharp.Compiler.Service.dll" />
+      <Reference Include="../../lib/fcs/FSharp.Compiler.Service.dll" />
+      <Reference Include="../../lib/fcs/FSharp.DependencyManager.Nuget.dll" />
```

Comment:
Is it normal that we have a new "dependency" here?

I asking in case this is something that has been forgotten.

Reply:
I won't be used in Transforms, so I will remove it.
If we were using vanilla FCS that ref will be there:
![image](https://github.com/fable-compiler/Fable/assets/2621499/34eb9cb0-2180-4aa8-a368-4c1f7647f9f2)

In this case, we are not using it, so I will remove it.